website/static/js/accountSettings.js

```diff
@@ -102,15 +102,20 @@ var UserProfileClient = oop.defclass({
         ).done(function (data) {
             ret.resolve(this.unserialize(data, profile));
         }.bind(this)).fail(function(xhr, status, error) {
-            $osf.growl('Error', 'User profile not updated. Please refresh the page and try ' +
+            if (error.toLowerCase() !== 'bad request') {
+                $osf.growl('Error', 'User profile not updated. Please refresh the page and try ' +
                 'again or contact <a href="mailto: support@cos.io">support@cos.io</a> ' +
                 'if the problem persists.', 'danger');
+            } else {
```

Comment:
It seems like invalid email is the special case, so maybe it would make sense to put that in the `if` part, and leave the more general (bad request) message for the `else` block?